exasol_script_languages_container_tool/lib/utils/logging_redirection.py

```diff
@@ -0,0 +1,40 @@
+import sys
+
+
+class _Tee(object):
+    # Helper object which redirects calls to stdout/stderr to itself, prints to a give file and to real stdout/stderr
+    def __init__(self, name, mode):
+        self.file = open(name, mode)
+        self.stdout = sys.stdout
+        self.stderr = sys.stderr
+        sys.stdout = self
+        sys.stderr = self
+
+    def get_file(self):
+        return self.file
+
+    def __del__(self):
+        sys.stdout = self.stdout
+        sys.stderr = self.stderr
+        self.file.close()
+
+    def write(self, data):
+        self.file.write(data)
+        self.stdout.write(data)
+
+    def flush(self):
+        self.file.flush()
+
+
+def create_docker_build_creator(task_creator):
```

Comment:
hm, I think, the name doesn't really fit. Maybe something like that `LogRedirectorTaskCreatorWrapper`